debian/control

```diff
@@ -8,5 +8,5 @@ Standards-Version: 4.1.2
 Package: wolkgateway
 Architecture: any
 Section: main
-Depends: ${shlibs:Depends}, ${misc:Depends}, libssl1.0.0 | libssl1.1 | libpoco-dev
+Depends: ${shlibs:Depends}, ${misc:Depends}, libssl1.0.0 | libssl1.1 | libssl-dev | libpoco-dev (>= 1.7)
```

Comment:
These two are probably needed in Build-Depends section and not during deployment